db/r2wiki

```diff
@@ -0,0 +1,33 @@
+R2PM_BEGIN
+
+R2PM_GIT "https://github.com/securisec/r2wiki"
+R2PM_DESC "[r2wiki] an internal wiki for r2"
+
+R2PM_DOC="
+usage: '$'wiki [-h] [-a] [-l] what_to_search_for
+
+positional arguments:
+  what_to_search_for  What to search for. Supports regex . Triple escape
+                      escapes. \\
+
+optional arguments:
+  -h, --help          show this help message and exit
+  -a                  Will match any of the words.
+  -l                  Show the corresponding r2wiki link
+  -u                  Update r2wiki with latest content
+
+Invoke from inside r2 shell with $wiki
+To update, use $wiki -u ''
+"
+
+R2PM_INSTALL() {
+	pip install --user Pygments
+	echo '$'wiki="#"'!'"pipe python ${R2PM_GITDIR}r2wiki/r2wiki.py" >> ~/.radare2rc || exit 1
+	echo "\n[+] r2wiki has been installed"
+}
+
+R2PM_UNINSTALL() {
+	rm -rf "${R2PM_GITDIR}"r2wiki
```

Comment:
Missing slash?

Reply:
In my test, the slash was coming up as //. so i took it out to keep it steady.